lib/AST/LocalizationFormat.cpp

```diff
@@ -113,15 +113,14 @@ YAMLLocalizationProducer::YAMLLocalizationProducer(llvm::StringRef filePath) {
   unknownIDs = std::move(yin.unknownIDs);
 }
 
-llvm::StringRef
-YAMLLocalizationProducer::getMessageOr(swift::DiagID id,
-                                       llvm::StringRef defaultMessage) const {
+llvm::Optional<llvm::StringRef>
+YAMLLocalizationProducer::getMessageOr(swift::DiagID id) const {
   if (diagnostics.empty())
-    return defaultMessage;
+    return llvm::None;
```

Comment:
Let's just add `using namespace llvm` and `using namespace swift` at the top of the file to avoid prefixing everything.